index.html

```diff
@@ -1,58 +1,246 @@
+    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+    crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+    integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+    crossorigin="anonymous"></script>
 </head>
 
 <body class="text-center">
-    <!-- navbar -->
-    <nav class="navbar fixed-top navbar-dark bg-dark">
-           
-            <button type="button" class="btn btn-dark btn-sm button3">button3</button>
-           
-            <button type="button" class="btn btn-dark btn-sm button">button2</button>
-            <button type="button" class="btn btn-dark btn-sm button">button1</button>
-            <img class="logo" src="images/logo.png" alt="logo">
-           
-      </nav>
-      <!-- sign-in -->
-      <div class="signin">
-        <form class="form-signin" onsubmit="return validateCheck()" novalidate>
-            <button type="button" class="btn btn-sm cross">X</button>
-            <div class="top-pad">
-                <h1 class="h3 mb-3 font-weight-normal">Please sign in</h1>
-                <div class="form-group row">
-                    <label for="staticEmail" class="col-sm-2 col-form-label">User</label>
-                    <div class="col-sm-10">
-                        <input type="text" class="form-control" id="user" placeholder="Username">
-                    </div>
-                  </div>
-                  <div class="form-group row">
-                    <label for="inputPassword" class="col-sm-2 col-form-label">Password</label>
-                    <div class="col-sm-10">
-                      <input type="password" class="form-control" id="userPassword">
-                    </div>
-                  </div>
-                <p class="p-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-                <button class="btn btn-lg btn-secondary btn-block" type="submit">Sign in</button>
+  <!-- navbar -->
+  <nav class="navbar fixed-top navbar-dark bg-dark">
+    <button type="button" class="btn btn-dark btn-sm button3">כפתור3</button>
+    <button type="button" class="btn btn-dark btn-sm button">כפתור2</button>
+    <button type="button" class="btn btn-dark btn-sm button" data-toggle="modal" onclick="clearWindow()"
```

Comment:
This can be risk worthy, if you enter your password and close the window and I'll come to your computer and open dev tools(F12) I can see in the HTML your password